src/test/scala/chiselTests/simulator/SimulatorSpec.scala

```diff
@@ -71,6 +71,29 @@ class SimulatorSpec extends AnyFunSpec with Matchers {
       assert(result === 12)
     }
 
+    it("reports failed expects correctly") {
+      val simulator = new VerilatorSimulator("test_run_dir/simulator/GCDSimulator")
+      val thrown = the [PeekPokeAPI.FailedExpectationException[_]] thrownBy {
+        simulator
+        .simulate(new GCD()) { module =>
+          import PeekPokeAPI._
+          val gcd = module.wrapped
+          gcd.io.a.poke(24.U)
+          gcd.io.b.poke(36.U)
+          gcd.io.loadValues.poke(1.B)
+          gcd.clock.step()
+          gcd.io.loadValues.poke(0.B)
+          gcd.clock.step(10)
+          gcd.io.result.expect(5)
+        }.result
+      }
+      thrown.getMessage must include regex "Observed value '12' != 5\\."
+      thrown.getMessage must include regex " @\\[src/test/scala/chiselTests/simulator/SimulatorSpec\\.scala \\d+:\\d+\\]"
+      thrown.getMessage must include regex "gcd\\.io\\.result\\.expect\\(5\\)"
+      // Not actually anchoring this so it's not extremely brittle, only somewhat.
+      thrown.getMessage must include regex "                    \\^"
```

Comment:
```suggestion
      thrown.getMessage must include("Observed value '12' != 5.")
      thrown.getMessage must include regex """ @\[src/test/scala/chiselTests/simulator/SimulatorSpec\.scala \d+:\d+\]"""
      thrown.getMessage must include("gcd.io.result.expect(5)")
      // Not actually anchoring this so it's not extremely brittle, only somewhat.
      thrown.getMessage must include("                    ^")
```

Very minor nits, but don't bother with regex when regular String include will do since it's more readable, and I also personally like using triple quotes for regex since I find having to escape `\` in regular Strings to build regexes unreadable.